demo/chatbot_demo/run.py

```diff
@@ -17,11 +17,12 @@ def chat(message, history):
     return history, history
 
 
+state = gr.Variable(value=[])
```

Comment:
using "state" is technically deprecated so I'm using `gr.Variable` here.

It makes the demo more verbose but it's more aligned with our current api. What do people think? Also had to modify the chatbot and real-time speech recognition guides.

Reply:
I think you can still use "state" shortcut in Interfaces. Is this demo not working for you as is @freddyaboulton? Because I just tested it on my end and it is working for me!

cc @aliabid94 as well

Reply:
Actually I think "state" shortcut is not deprecated in `Interfaces`. Is this demo not working for you as is @freddyaboulton? Because I just tested it on my end and it is working for me!

cc @aliabid94 as well